pom.xml

```diff
@@ -4,17 +4,19 @@
   <parent>
     <groupId>org.jvnet.hudson.plugins</groupId>
     <artifactId>analysis-pom</artifactId>
-    <version>1.59</version>
+    <version>1.63</version>
   </parent>
 
   <artifactId>pmd</artifactId>
   <packaging>hpi</packaging>
   <name>PMD Plug-in</name>
   <version>3.42-SNAPSHOT</version>
-  <url>http://wiki.jenkins-ci.org/x/GAAHAQ</url>
-  <description>This plug-in generates the trend report for PMD, an
-    open source static code analysis program.
-  </description>
+  <url>https://wiki.jenkins-ci.org/display/JENKINS/PMD+Plugin</url>
+  <description>This plug-in generates the trend report for PMD, an open source static code analysis program.</description>
```

Comment:
Found, seems permalink is the right way.
(imho  even if url changed you can change it for next releases).